clients/android/NewsBlur/assets/dark_reading.css

```diff
@@ -1,22 +1,24 @@
-body, span {
+body, span, table, td, p, div {
 	background-color: #1A1A1A !important;
 	color: #FFF !important;
 }
 
 a, a span {
 	color: #319DC5 !important;
+	background-color: #1A1A1A !important;
 }
 
 a:visited, a:visited span {
 	color: #319DC5 !important;
+	background-color: #1A1A1A !important;
 }
 
-code {
+pre, blockquote, code {
     background-color: #4C4C4C;
 }
 
-pre, blockquote {
-    background-color: #4C4C4C;
+blockquote > p, pre > span, code span {
```

Comment:
pre is now handled in the previous selector.

On Mon, 7 Mar 2016 19:46 Samuel Clay, <notifications@github.com> wrote:

> In clients/android/NewsBlur/assets/dark_reading.css
> <https://github.com/samuelclay/NewsBlur/pull/892#discussion_r55260259>:
>
> >      background-color: #4C4C4C;
> >  }
> >
> > -pre, blockquote {
> > -    background-color: #4C4C4C;
> > +blockquote > p, pre > span, code span {
>
> What about just pre?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/samuelclay/NewsBlur/pull/892/files#r55260259>.
>